src/main/java/org/launchcode/techjobs/persistent/controllers/HomeController.java

```diff
@@ -1,20 +1,34 @@
 package org.launchcode.techjobs.persistent.controllers;
 
+import org.launchcode.techjobs.persistent.models.Employer;
 import org.launchcode.techjobs.persistent.models.Job;
+import org.launchcode.techjobs.persistent.models.Skill;
+import org.launchcode.techjobs.persistent.models.data.EmployerRepository;
+import org.launchcode.techjobs.persistent.models.data.JobRepository;
+import org.launchcode.techjobs.persistent.models.data.SkillRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.validation.Errors;
 import org.springframework.web.bind.annotation.*;
 
 import javax.validation.Valid;
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Optional;
 
 /**
  * Created by LaunchCode
  */
 @Controller
 public class HomeController {
 
+    @Autowired
+    private EmployerRepository employerRepository;
+    @Autowired
+    private JobRepository jobRepository;
+    @Autowired
+    private SkillRepository skillRepository;
     @RequestMapping("")
     public String index(Model model) {
```

Comment:
On your home page, you can pass in jobs to the template using `model.addAttribute()` to render them on that page.

```
    @RequestMapping("")
    public String index(Model model) {

        model.addAttribute("title", "My Jobs");
        model.addAttribute("jobs", jobRepository.findAll());
        return "index";
    }
```